proxy/HostStatus.h

```diff
@@ -46,7 +49,14 @@ struct HostStatRec_t {
   unsigned int down_time; // number of seconds that the host should be down, 0 is indefinately
 };
 
-const std::string stat_prefix = "host_status.";
+static const std::string stat_prefix = "proxy.process.host_status.";
+static const std::string MANUAL      = "manual";
+
+static const std::vector<std::string> reasons = {
```

Comment:
`std::array` is better IMHO for constant arrays.

Reply:
I refactored and am using a struct now with char * array.